ccxt.d.ts

```diff
@@ -272,12 +332,53 @@ declare module 'ccxt' {
         twofa: boolean;// two-factor authentication
         substituteCommonCurrencyCodes: boolean;
         timeframes: any;
-        has: { [what: string]: any }; // https://github.com/ccxt/ccxt/pull/1984
+        has: Dictionary<any>; // https://github.com/ccxt/ccxt/pull/1984
         balance: object;
         orderbooks: object;
         orders: object;
         trades: object;
         userAgent: { 'User-Agent': string } | false;
+        limits: { amount: MinMax, price: MinMax, cost: MinMax };
```

Comment:
I'm not sure why Exchange has this property. For binance it returns:
```js
{
  amount: { min: undefined, max: undefined },
  price: { min: undefined, max: undefined }, 
  cost: { min: undefined, max: undefined }   
}
```

Reply:
@omidkrad it's the exchange-wide defaults that get extended and overrided for each particular market. Some exchanges don't have min limits defined anywhere in their API, therefore we have to define their limits for the entire exchange, and that structure gets inherited by every market. Binance does have min limit info in the market APIs themselves, therefore the values of this exchange-wide property is undefined – it gets defined upon `fetchMarkets()/loadMarkets()`.